frontend/src/types/settings.ts

```diff
@@ -18,7 +32,7 @@ export type Settings = {
   ENABLE_DEFAULT_CONDENSER: boolean;
   ENABLE_SOUND_NOTIFICATIONS: boolean;
   USER_CONSENTS_TO_ANALYTICS: boolean | null;
-  PROVIDER_TOKENS: Record<Provider, string>;
+  PROVIDER_TOKENS: Record<Provider, ProviderTokenDataGet | string>;
```

Comment:
Can we enforce this to always be `ProviderTokenData` instead of `string

Also the backend route can end empty token for `GET` requests, instead of making `ProviderTokenDataPost` and `ProviderTokenDataGet`